tests/test-rest-tags-controller.php

```diff
@@ -224,6 +224,45 @@ public function test_get_items_post_args() {
 		$this->assertEquals( 'DC', $data[0]['name'] );
 	}
 
+	public function test_get_terms_post_args_paging() {
+		$post_id = $this->factory->post->create();
+		$tag_ids = array();
+
+		for ( $i = 0; $i < 30; $i++ ) {
```

Comment:
I totally agree and I will keep that in mind for the future, but I was just follow the other [test](https://github.com/WP-API/WP-API/blob/develop/tests/test-rest-tags-controller.php#L344) in this same file that created 50 post tags ;)